UI/MainForm.cs

```diff
@@ -332,6 +333,37 @@ private void CheckForWasmModuleUpdate()
             
         }
 
+        /// <summary>
+        /// Returns a list of connected USB drives that are supported with MobiFlight and are in flash mode already,
+        /// as opposed to being connected as COM port.
+        /// </summary>
+        /// <returns>The list of connected USB drives supported by MobiFlight.</returns>
+        private static List<MobiFlightModuleInfo> FindConnectedUsbDevices()
```

Comment:
This is the method used to find devices that are already in BOOTSEL mode when MobiFlight runs. Seems very strange to have it in a `MainForm.cs` file but I didn't know anywhere else better to put it.

Reply:
A better place is MobiflightCache because it contains a list of all modules.

Reply:
Moved to MobiFlightCache.cs as a static method.